lib/src/utils/menu.dart

```diff
@@ -55,9 +55,16 @@ class PullDownMenu extends StatelessWidget {
             borderRadius: theme.borderRadius!,
             child: FadeTransition(
               opacity: animation,
-              child: _MenuBody(
-                width: theme.width!,
-                children: items,
+              child: SizedBox(
+                width: theme.width,
+                child: SizeTransition(
+                  axisAlignment: -1,
```

Comment:
Does this **-1** work when menu opens in upwards direction like with `PullDownMenuPosition.above`

Reply:
Yep, it works like a native platform

https://user-images.githubusercontent.com/42691184/221403215-d54e7e61-10f9-4afe-8117-72b02736c187.mp4

Reply:
Dope thx